### ISSUE TITLE
Updating Examples

### ISSUE THREAD

**Issue by nvwebd** [maxmantz/redux-oidc#151]
Hi.

I wanted to open this Issue so I get some info before I start updating the examples - for guidelines and additional info.

What I wanted to do is create a new example app showing implementation without redux-router and also a better hook integration.

Let me know if you would consider this a + to PR or not.

Best from Vienna.

Nejc

**Comment by eloekset** [comment]
I'm very interested in a working sample app for React IODC with or without redux. Have you got something up an running, @nvwebd?

**Comment by nvwebd** [comment]
@eloekset yeah I have something up and running - with Redux - for without I still have to do something, but I didn't manage to find the time.

**Comment by maxmantz** [comment]
Hi!

If you have an updated example I would be happy to link it to the docs.

**Comment by davidkhierl** [comment]
I would love to see some example for the `SignoutCallbackComponent` also

**Comment by henriquels25** [comment]
I created a new example project using the library: https://github.com/henriquels25/redux-oidc-sample

**Comment by henriquels25** [comment]
I created a new example project using the library: 
https://github.com/henriquels25/redux-oidc-sample

@maxmantz, do you think we can link it to the docs?

If someone has suggestions, I can work on improving the example application.

**Comment by aleksandervalle** [comment]
Thanks for the new example @henriquels25. 

I expected [CallbackComponent](https://github.com/maxmantz/redux-oidc/blob/master/src/CallbackComponent.js) to populate redux store with user, but it doesn't. Am I supposed to dispatch `USER_FOUND` myself in the successCallback. I see neither the existing example nor @henriquels25 new example do this, or am I missing something?

**Comment by aleksandervalle** [comment]
Never mind! It's done by the OidcProvider upon user load event.
